### PR TITLE
Alert users pymysql is not required for external MariaDB

### PR DESCRIPTION
**Pull request by omegacore**
Please include this change so others don't have to waste their time. I only realized this because I started reading issues around pymysql and getting ready to open an PR for pymysql to be included in the docker image.

### CODE REVIEW

File: source/_integrations/recorder.markdown [hunk 1]
```diff
@@ -148,6 +148,18 @@ Some installations of MariaDB/MySQL may require an ALTERNATE_PORT (3rd-party hos
...
+| Database engine | `db_url`                                                 |
+| :---------------|:---------------------------------------------------------|
+| MariaDB         | `mysql://SERVER_IP/DB_NAME?charset=utf8`                 |
+| MariaDB         | `mysql://user:password@SERVER_IP/DB_NAME?charset=utf8`   |
+
+`pymysql` is not included in the base docker image, and is not necessary for this to work.
+
+</div>
+
+<div class='note'>
```
Comment (fabaff) [inline source/_integrations/recorder.markdown:156]: This could be part of the "Custom database engines" table in the same way as PostgreSQL is covered..

**Review by fabaff — DISMISSED** [review]

**Comment by fabaff** [comment]
This should go to the `current` branch.

**Comment by omegacore** [comment]
Does this still need rebasing? @klaasnicolaas

**Comment by klaasnicolaas** [comment]
Yes you still need to rebase this PR to the `current` branch

**Comment by frenck** [comment]
Rebased PR onto the latest `current` branch and changed the target branch for the PR.

**Review by frenck — APPROVED** [review]
Thanks, @omegacore! 👍 

Happy New Year 🍾

**Pull request merged**